src/components/_input.scss

```diff
@@ -42,6 +42,7 @@
     transition-duration: 150ms;
     transition-property: border-color;
     appearance: none;
+    -moz-appearance: textfield; // stylelint-disable-line
```

Comment:
why is this needed?

Reply:
Firefox doesn’t respect the `::webkit-inner-spin-button` 🤷

Reply:
And setting it to `-moz-appearance: none` still shows it too 🤷 🤷‍♂️ 🤷‍♀️